docs/content/developers/konnector/controllers/cluster/apiserviceexport.md

```diff
@@ -0,0 +1,93 @@
+# APIServiceExports
+
+The APIServiceExport controller watches `APIServiceExports` and the referenced `CustomResourceDefinitions` (CRDs) in the **provider cluster**.
+
+It is responsible for:
+
+* Ensuring the existence and validity of `APIServiceExports`.
+* Managing the lifecycle of `APIServiceExports` by starting and stopping spec and status syncers and controllers.
+* Checking `APIServiceBinding` condition and setting `ConsumerInSync` condition in `APIServiceExport`
+* Copying conditions from the referenced CRDs to the `APIServiceExport` status.
+
+## Overview
+
+```mermaid
```

Comment:
am curious, is this auto-generated in some way or manual by studying the code?

Reply:
@sttts it is manual, but I would love to generate it as well